# Don't let gap build it's own Normaliz, use the EB version instead
# wpoely86@gmail.com
diff -ur 4.9.3-intel-2018b.orig/pkg/NormalizInterface-1.0.2/build-normaliz.sh 4.9.3-intel-2018b/pkg/NormalizInterface-1.0.2/build-normaliz.sh
--- 4.9.3-intel-2018b.orig/pkg/NormalizInterface-1.0.2/build-normaliz.sh	2018-10-16 15:26:01.350316915 +0200
+++ 4.9.3-intel-2018b/pkg/NormalizInterface-1.0.2/build-normaliz.sh	2018-10-16 15:24:30.353651528 +0200
@@ -12,6 +12,8 @@
 #  make
 #
 
+exit 0
+
 if [ "$#" -ge 1 ]; then
     GAPDIR=$1
     shift
